front/css.php

```diff
@@ -38,11 +38,15 @@
 use Glpi\Application\Environment;
 use Glpi\UI\ThemeManager;
 
-// Main CSS compilation requires about 70MB of memory.
-// Ensure to have enough memory to not reach memory limit.
-$max_memory = 96;
-if (Toolbox::getMemoryLimit() < ($max_memory * 1024 * 1024)) {
-    ini_set('memory_limit', sprintf('%dM', $max_memory));
+use function Safe\ini_set;
+use function Safe\preg_match;
+
+if (preg_match('~^css/glpi(\.scss)?$~', $_GET['file'] ?? '') === 1) {
+    // Ensure to have enough memory to not reach memory limit.
+    $max_memory = Html::MAIN_SCSS_COMPILATION_REQUIRED_MEMORY;
+    if (Toolbox::getMemoryLimit() < ($max_memory * 1024 * 1024)) {
+        ini_set('memory_limit', sprintf('%dM', $max_memory));
```

Comment:
You should care about cases when ini_set() returns false, it may happen.
Throwing an Exception, probably.